backend/bankaccounts/src/main/resources/bankaccounts-persister.yml

```
@@ -0,0 +1,400 @@
+          basePath: $.bankAccounts.*.bankAccountDetails.*.bankBranchIdentifier
+          jsonMaps:
+
+            - jsonPath: $.bankAccounts.*.bankAccountDetails.*.bankBranchIdentifier.type
+
+            - jsonPath: $.bankAccounts.*.bankAccountDetails.*.bankBranchIdentifier.code
+
+            - jsonPath: $.bankAccounts.*.bankAccountDetails.*.bankBranchIdentifier.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.bankAccounts.*.bankAccountDetails.*.bankBranchIdentifier.id
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)